frontend/src/tests/unit/validators.test.js

```diff
@@ -100,55 +112,28 @@ describe('Email validator tests:', () => {
     expect(validatorMessage1).toBe(FIELD_REQUIRED)
     expect(validatorMessage2).toBe(FIELD_REQUIRED)
   })
-  it('should return INCORRECT_EMAIL error for bad email string', () => {
-    // given
-    const exampleEmail1 = 'testEmail'
-    const exampleEmail2 = 'testEmail@'
-    const exampleEmail3 = 'testEmail@test.'
-    const exampleEmail4 = 'test@test.com'
-    const exampleEmail5 = 'test@test.pl'
-    const exampleEmail6 = 'test@agh.edu.pl.test.pl'
-    const exampleEmail7 = '@agh.edu.pl'
-    const exampleEmail8 = '123456@agh.edu.pl'
-
+  it.each(incorrectEmails)('should return INCORRECT_EMAIL error for %s', (email) => {
     // when
-    const validatorMessage1 = validateEmail(exampleEmail1, AccountType.STUDENT)
-    const validatorMessage2 = validateEmail(exampleEmail2, AccountType.PROFESSOR)
-    const validatorMessage3 = validateEmail(exampleEmail3, AccountType.PROFESSOR)
-    const validatorMessage4 = validateEmail(exampleEmail4, AccountType.STUDENT)
-    const validatorMessage5 = validateEmail(exampleEmail5, AccountType.PROFESSOR)
-    const validatorMessage6 = validateEmail(exampleEmail6, AccountType.STUDENT)
-    const validatorMessage7 = validateEmail(exampleEmail7, AccountType.PROFESSOR)
-    const validatorMessage8 = validateEmail(exampleEmail8, AccountType.STUDENT)
+    const validatorMessage1 = validateEmail(email, AccountType.STUDENT)
+    const validatorMessage2 = validateEmail(email, AccountType.STUDENT)
```

Comment:
Czy tutaj 2 razy nie wywołuje się teraz to samo?

Reply:
miało być w drugim PROFESSOR, mój błąd :/